src/client/features/view/index.js

```diff
@@ -89,8 +99,33 @@ class View extends React.Component {
   performLayout(layoutName) {
     this.setState({layout: layoutName});
     const cy = this.state.cy;
+
+    if (layoutName === lo.humanLayoutName) {
+      const layoutJSON = this.state.layoutJSON;      
+      let options = {
+        name: 'preset',
+        positions: node => layoutJSON[node.id()],
+        animate: true,
+        animationDuration: 500
+      };
+      cy.nodes('[class="complex"], [class="complex multimer"]').filter(node => node.isExpanded()).collapse();
+      cy.layout(options).run();
+      return;
+    }
+
     cy.nodes('[class="complex"], [class="complex multimer"]').filter(node => node.isExpanded()).collapse();
-    cy.layout(lo.layoutMap.get(layoutName)).run();
+    let layout = cy.layout(lo.layoutMap.get(layoutName));
+    let that = this;
+    layout.pon('layoutstop').then(function() {
+      if (that.props.admin && layoutName !== lo.humanLayoutName) {
+        let posObj = {};
+        for (let i = 0; i < 25; i++) {
```

Comment:
What is the purpose of iterating 25 times here?

Reply:
Accidentally left in a hard coded value. Just fixed it.